File: MASICBrowser/modMASICBrowser.vb
Module modMASICBrowswer

    'Private objSpectrum As CWSpectrumDLL.SpectrumClass

    Public Sub Main()

        Dim objBrowserForm As frmBrowser

        Try
            'objSpectrum = New CWSpectrumDLL.SpectrumClass
            'objSpectrum.ShowSpectrum()

            objBrowserForm = New frmBrowser
            objBrowserForm.ShowDialog()

        Catch ex As Exception
            MsgBox("Unable to initialize the CW Spectrum DLL.  Ending program." & vbCrLf & ex.Message, MsgBoxStyle.Exclamation Or MsgBoxStyle.OKOnly, "Missing DLL")
        End Try

    End Sub

    Public Sub ShowSICSpectrum()
        'objSpectrum.ShowSpectrum()
    End Sub
End Module

File: MASICBrowser/modMASICBrowser.cs
using System;

static class modMASICBrowswer
{

    // private CWSpectrumDLL.SpectrumClass objSpectrum;

    public static void Main()
    {
        frmBrowser objBrowserForm;

        try
        {
            // objSpectrum = new CWSpectrumDLL.SpectrumClass();
            // objSpectrum.ShowSpectrum();

            objBrowserForm = new frmBrowser();
            objBrowserForm.ShowDialog();
        }
        catch (Exception ex)
        {
            MessageBox.Show("Unable to initialize the CW Spectrum DLL.  Ending program." + Environment.NewLine + ex.Message, "Missing DLL", MessageBoxButtons.Ok, MessageBoxIcons.Exclamation);
        }
    }

    public static void ShowSICSpectrum()
    {
        // objSpectrum.ShowSpectrum();
    }
}
